<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>prefs</class>
 <widget class="QWidget" name="prefs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="themePref">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>221</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="themeLab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Theme:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="fmtPref">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>221</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Imperial</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Metric</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Standard (Kelvin)</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="fmtLab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Format: </string>
   </property>
  </widget>
  <widget class="QLabel" name="prefLab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="outputLab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Console Output:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="outputPref">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>221</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Info</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Errors only</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Debug</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
